doc/src/manual/functions.md

```diff
@@ -181,20 +182,20 @@ syntaxes:
 
 ```jldoctest
 julia> x -> x^2 + 2x - 1
-#1 (generic function with 1 method)
+(::#1) (generic function with 1 method)
```

Comment:
Similarly, this seems to be for v0.6, while this document is for v0.7.

Reply:
@vtjnash What happened to my example? is that ok?